glustercli/cli/parsers.py

```diff
@@ -9,6 +9,120 @@ class GlusterCmdOutputParseError(Exception):
     pass
 
 
+HEALTH_UP = "up"
+HEALTH_DOWN = "down"
+HEALTH_PARTIAL = "partial"
```

Comment:
What is the difference between partial and degraded?

Reply:
partial - some bricks are down but volume is up not degraded.
degraded - some subvols are down, volume is not fully available